pkg/cmd/start/start_pipeline.go

```diff
@@ -204,7 +204,7 @@ func (o *StartPipelineOptions) createMetaPipeline(jobName string) error {
 		return errors.Wrap(err, "failed to create JX client")
 	}
 
-	sr, err := kube.FindSourceRepository(jxClient, ns, owner, repo)
+	sr, err := kube.FindSourceRepository(jxClient, ns, owner, repo, "", false)
```

Comment:
Why the empty string here? Having to pass empty strings is always confusing imo.

Reply:
Changing to have a `FindSourceRepositoryWithoutProvider` function. It's moments like this that I miss method overloading in Java. =)